tests/Vue/Components/MismatchRow.spec.js

```diff
@@ -104,6 +104,118 @@ describe('MismatchesRow.vue', () => {
         expect( linkUrl ).toEqual( mismatch.import_meta.external_source_url );
     });
 
+    it('truncates the upload info description and adds "see full description" link when longer than 100 chars ', () => {
+        const mismatch = {
+            property_label: 'Hey hey',
+            wikidata_value: 'Some value',
+            external_value: 'Another Value',
+            review_status: 'pending',
+            import_meta: {
+                user: {
+                    username: 'some_user_name'
+                },
+                external_source: 'some external source',
+                external_source_url: 'http://www.whatever.com',
+                created_at: '2021-09-23',
+                description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
+                Suspendisse viverra ut quam eget congue. Nulla accumsan hendrerit eleifend. 
+                Donec eget tempor metus. Cras sit amet pellentesque eros. Pellentesque mattis 
+                sed justo ac commodo. Proin auctor lectus congue ligula lacinia dignissim.`
+            },
+        };
+
+        const wrapper = mount(MismatchRow, {
+            propsData: { mismatch },
+            mocks: {
+                // Mock the banana-i18n plugin dependency
+                $i18n: key => key
+            }
+        });
+
+        const desiredTruncateLength = 100;
```

Comment:
I wonder if we should keep think in some external file rather than specify it both here and in the `vue` file?

Reply:
maybe, some global config or env file.  agree.

Reply:
maybe that's a good question for Lydia before she leaves for two weeks.